### DISCUSSION TITLE
Добавить файлы через временную таблицу

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#308] [Как это сделать?] [answered]
На форуме видел, что многие пользуются временными таблицами для добавления файлов. Но я не очень понимаю, как это работает.

Задача у меня вот какая:
Для добавления новых файлов есть временная таблица с полем в хедере **h_file**, с типом файл + множественные значения.

Нужно, чтобы добавленные в **h_file** файлы, будь то один или несколько, добавлялись каждый в отдельную строку в таблицу ниже, а поле **h_file** сразу чтобы очищалось после этого действия. И так сколько угодно раз, и сколько угодно файлов можно добавлять.

А потом по нажатию отдельной кнопки ОК, чтобы все строки из этой таблицы вместе с файлами добавлялись в основную простую таблицу ФАЙЛЫ.

Я так понимаю, что ещё будут создаваться временные файлы с пометкой !tmp!, которые нужно очищать. Можно ли это делать действием из этой таблицы или только через крон?

Спасибо

**Comment by erpic-ru** [comment, marked as answer]
[Вот тут](https://github.com/totumonline/totum-ru-issues-and-discussions/discussions/276) обсуждали как сделать

**Reply by mihail-cornev** [reply]
Спасибо, как-то я пропустил этот топик.
